src/Devito.jl

```diff
@@ -187,8 +187,26 @@ function Base.copy!(dst::DevitoMPISparseArray, src::Array)
     dst
 end
 
+#
+# BasicDimension
+#
+abstract type BasicDimension end
```

Comment:
I think Julia convention would be to call this either `Dimension` or `AbstractDimension`.  See, for example, the `AbstractFloat` type.   However, if  Devito uses the name `BasicDimension`, then I guess that's a different story :)

Reply:
Changing it to Dimension.